<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TrustedTest</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>App.png</normaloff>App.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1001</width>
     <height>601</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: rgb(150, 238, 238);
}
</string>
   </property>
   <widget class="QPushButton" name="but_assess">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>110</y>
      <width>180</width>
      <height>140</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Take a quick test</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
	background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
background-color: rgb(214, 238, 205);
}</string>
    </property>
    <property name="text">
     <string>Assessment Test</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_condoc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>251</y>
      <width>180</width>
      <height>140</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Contact details of doctor</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
background-color: rgb(214, 255, 205);
}</string>
    </property>
    <property name="text">
     <string>Contact Doctor</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_news">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>251</y>
      <width>180</width>
      <height>140</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Latest covid news/info</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
	background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
background-color: rgb(214, 238, 205);
}</string>
    </property>
    <property name="text">
     <string>News</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_abtus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Our story</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 15pt &quot;Calibri&quot;;
	background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 15pt &quot;Calibri&quot;;
background-color: rgb(214, 238, 205);
}</string>
    </property>
    <property name="text">
     <string>About Us</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_stat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>180</width>
      <height>140</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Contact details of doctor</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
	background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 18pt &quot;Calibri&quot;;
background-color: rgb(214, 238, 205);
}</string>
    </property>
    <property name="text">
     <string>Test Statistics</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>1120</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>RNA.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="but_log">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Latest covid news/info</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:10px;
font: 15pt &quot;Calibri&quot;;
	background-color: rgb(195, 250, 177);
}
QPushButton:hover{
border-radius:10px;
font: 15pt &quot;Calibri&quot;;
background-color: rgb(214, 238, 205);
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>but_abtus</zorder>
   <zorder>but_condoc</zorder>
   <zorder>but_assess</zorder>
   <zorder>but_stat</zorder>
   <zorder>but_log</zorder>
   <zorder>but_news</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
